pkg/api/login_test.go

```diff
@@ -613,12 +608,17 @@ func (r *loginHookTest) LoginHook(loginInfo *models.LoginInfo, req *models.ReqCo
 func TestLoginPostRunLokingHook(t *testing.T) {
 	sc := setupScenarioContext(t, "/login")
 	hookService := &hooks.HooksService{}
+	testUser := &models.User{
+		Id:    42,
+		Email: "",
+	}
 	hs := &HTTPServer{
 		log:              log.New("test"),
 		Cfg:              setting.NewCfg(),
 		License:          &licensing.OSSLicensingService{},
 		AuthTokenService: auth.NewFakeUserAuthTokenService(),
 		HooksService:     hookService,
+		MultiLDAPService: &multildap.MultiLDAPmock{UserInfo: testUser},
```

Comment:
So, what if you do instead:

```
ldapMock := &multildap.MultiLDAPmock{UserInfo: testUser}
hs := &HTTPServer{ .... MultiLDAPService: ldapMock}
...
ldapMock.UserInfo = c.authUser
ldapMock.AuthModule = c.authModule
```
to avoid blind type casts?